san/transform.py

```diff
@@ -11,9 +11,11 @@
 QUERY_PATH_MAP = {
     'eth_top_transactions': ['ethTopTransactions'],
     'eth_spent_over_time': ['ethSpentOverTime'],
-    'token_top_transactions':['tokenTopTransactions']
+    'token_top_transactions':['tokenTopTransactions'],
+    'get_metric':['timeseriesData']
 }
 
+V2_METRIC_QUERIES = ['average_price', 'daily_active_addresses', 'mvrv_usd_20y']
```

Comment:
Why these queries only?
`average_price` should be removed, too

Reply:
And we need to add much more metrics here: https://github.com/santiment/sanbase2/blob/master/lib/sanbase/clickhouse/metric/available_v2_metrics.json

Reply:
This is temporary, just wanted to see if it should be implemented like this or it should be changed. Will add them in a bit